tests/CMakeLists.txt

```diff
@@ -24,7 +25,7 @@ math(EXPR PARALLEL_COUNT "2 * ${NUM_CPU}")
 
 add_custom_target(
   check
-  COMMAND ${CMAKE_CTEST_COMMAND} -j${PARALLEL_COUNT} -T test --output-on-failure
+  COMMAND ${CMAKE_CTEST_COMMAND} -j${PARALLEL_COUNT} -T test --output-on-failure -C Debug
```

Comment:
Why do we need to use Debug here?

Reply:
The non-ninja builder requires a build-type. Otherwise the tests don't work on Windows.